### PR TITLE
CCD-1318 Add new dashboard for CCD nightly jobs

### PR DESCRIPTION
**Pull request by owen-delaney-cgi**
### Change description ###
Create new dashboard for nightly builds
Remove nightly builds from existing dashboard
update develop branch build of some apps

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml [hunk 1]
```diff
@@ -99,10 +99,16 @@ spec:
...
                     includeRegex: >-
-                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api).+\/master
+                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api|.HMCTS_Nightly.*CDM.*).+\/master
                     name: CDM
                     recurse: true
                     title: CDM Builds Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_Nightly.*CDM.*\/(?!.*ccd-admin-web-api).+\/master
+                    name: CDM-Nightly
```
Comment (MSancaktutar) [inline apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml:109]: "CDM Nightly" for consistency with "CDM DEV" dashboard,

File: apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml [hunk 2]
```diff
@@ -99,10 +99,16 @@ spec:
                     title: RPX-PR
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api).+\/master
+                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api|.HMCTS_Nightly.*CDM.*).+\/master
                     name: CDM
                     recurse: true
                     title: CDM Builds Dashboard
```
Comment (MSancaktutar) [inline apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml:105]: "CDM Master Builds Dashboard" to make this now more specific.

File: apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml [hunk 3]
```diff
@@ -99,10 +99,16 @@ spec:
...
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_Nightly.*CDM.*\/(?!.*ccd-admin-web-api).+\/master
+                    name: CDM-Nightly
+                    recurse: true
+                    title: CDM Nightly Builds Dashboard
                 - buildMonitor:
                     includeRegex: >-
                       ^HMCTS_.*CDM.*\/(.*ac-manage-case-assignment.*PR-272|.*ccd-admin-web.*PR-325|.*ccd-case-activity-api.*PR-256|.*ccd-case-print-service.*PR-260|.*ccd-data-store-api.*PR-1260|.*ccd-definition-store-api.*PR-575|.*ccd-user-profile-api.*PR-399|.*cpo-case-payment-orders-api.*PR-54|.*ccd-api-gateway.*PR-353|.*ccd-case-activity-api.*PR-256|.*ccd-case-print-service.*PR-260|.*ccd-case-management-web.*PR-1043)
```
Comment (MSancaktutar) [inline apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml:114]: Could you please also check the fullness and correctness of develop builds? Some apps with a develop branch doesn't appear under that dashboard. The PR numbers for develop into master of those repos may have just changed to a new PR recently. CCD Case Management Web is one of those, for example.

**Review by timja — APPROVED** [review]

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**